### ISSUE TITLE
Giving style to intermediary div

### ISSUE THREAD

**Issue by johanlajili** [httptoolkit/react-reverse-portal#23]
Hi, thanks for your lib it's been super helpful for us.
We've been using it to render a 3D app in different pages of our react application.

However for this to work well, we need the 3D to take 100% of the space of its container. This is broken in reverse-portal because of the intermediary div you create (document.createElement('div')), which does not have any style, classname, or way to inject those.
We've been using nth-child(1) in React as a work around, but moving to preact, it seems it's not always reliable (I believe sometimes the placeholder div remains in the DOM). Could you add a way for us to inject either a style or class name to that div?

**Comment by pimterry** [comment]
That makes sense, I'm open to this. I think the easiest option is some kind of `attributes` option for the create*PortalNode methods. Then you could do this like:

```javascript
const portalNode = portals.createHtmlPortalNode({
	attributes: { 'class': 'my-class-name' }
});
```

I think this would have to be DOM attributes, _not_ React props, since the wrapper node is always built with the raw DOM API. That lets you configure anything you like though, from classes to styles to anything else.

Would that work for you? I'm happy to review and merge a PR to add that, if so.

**Comment by johanlajili** [comment]
Yeah works for me, we'll open a PR in the coming week thanks.

**Comment by johanlajili** [comment]
Closing as solved by #28
